torchx/test/torchx/nx_test.exs

```diff
@@ -420,18 +420,6 @@ defmodule Torchx.NxTest do
       )
     end
 
-    test "dot currently raises when using batching" do
```

Comment:
The rest of the dot related tests here could also be removed as they are testing the same thing as the doctests, so now that those are restored they are not needed.